editor.planx.uk/src/@planx/components/MapAndLabel/model.ts

```diff
@@ -2,15 +2,19 @@ import { MoreInformation, parseMoreInformation } from "../shared";
 
 export interface MapAndLabel extends MoreInformation {
   fn: string;
-  title?:string;
-  description?:string;
+  title: string;
+  description?: string;
+  drawColour: string;
+  drawType?: "Polygon" | "Point";
```

Comment:
Is there a reason `drawType` is optional but `drawColour` is required here? Can they both be required with smart defaults (which looks like how you've set them up below!)?